kratos/sources/dem_variables.cpp

```diff
@@ -43,15 +43,15 @@ namespace Kratos
     KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS( DAMP_FORCES )
     KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS( AUX_VEL )
     KRATOS_CREATE_VARIABLE( Vector, NEIGHBOURS_IDS_DOUBLE )
-    KRATOS_CREATE_VARIABLE( vector<double>, BASSET_HISTORIC_INTEGRANDS )
-    KRATOS_CREATE_VARIABLE( vector<double>, HINSBERG_TAIL_CONTRIBUTIONS )
+    KRATOS_CREATE_VARIABLE( DenseVector<double>, BASSET_HISTORIC_INTEGRANDS )
+    KRATOS_CREATE_VARIABLE( DenseVector<double>, HINSBERG_TAIL_CONTRIBUTIONS )
     KRATOS_CREATE_VARIABLE( Vector, PARTICLE_ROTATE_SPRING_FAILURE_TYPE )

```

Comment:
Vector -> DenseVector<double> ?

Reply:
You are right. I'm not coherence in changing them.... They are equivalent but I can change them to be coherent.